docs/toc.yml

```diff
@@ -41,7 +41,7 @@
     href: overview/set-up-your-dev-environment.md
     displayName: Excel, Word, OneNote, Project, PowerPoint, Outlook
 
-- name: Core concepts
+- name: Development process
```

Comment:
Is it weird to have "Dev process overview" and then "Dev process" underneath it? Curious on your opinion on this one :)

Reply:
Discussed offline, here is a summary for record: I do feel conflicted about using variations of 'develop' at so many levels, but I also think that "Development process" is a clearer high-level node label than "Core concepts" so I continue to recommend it at this point. Hoping we can make further clarifications in the future, perhaps by employing another synonym for 'develop' in some TOC locations.